bot/seasons/valentines/poem_generator.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import random
+from typing import Dict, List
+
+import aiohttp
+import markovify
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class PoemGenerator:
+    SOURCES = [
+        "shakespeare corpus.txt"
```

Comment:
Spaces in resource file names are generally discouraged, and other resources already avoid them also, making it not really match the current project style. Could this be renamed without it?